applications/osb-portal/src/pages/WorkspacePage.tsx

```diff
@@ -152,7 +152,12 @@ export const WorkspacePage = (props: any) => {
   React.useEffect(() => {
     WorkspaceService.getWorkspace(parseInt(workspaceId, 10)).then((ws) => {
       setWorkspace(ws);
-    });
+      props.setError(null);
+    },
+    (error) => {
+      const errorMessage = `Oops! This workspace could not be accessed. (Code: ${error.status})`;
```

Comment:
I don't like very much to give error codes that the average user does not understand. I would either specify the the error as unauthorized for 401 or just leave it generic.

Reply:
Hrm, the idea here is that if the user does contact us, we know what the error was because 401 is not the only error they get. So, I can hard code a switch statement there that checks the error code and gives the user the English interpretation if that works better? Unless there's a better way to automatically translate the error codes into English for the user?

Reply:
Sentry is already doing the tracking for us, we can add any additional information directly there.

Reply:
cool, removed error code.